indexing-service/src/main/java/org/apache/druid/indexing/common/actions/UpdateLocationAction.java

```diff
@@ -23,11 +23,10 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.core.type.TypeReference;
-import com.google.common.base.Optional;
 import org.apache.druid.indexer.TaskLocation;
 import org.apache.druid.indexing.common.task.Task;
-import org.apache.druid.indexing.overlord.TaskRunner;
 
+@Deprecated
```

Comment:
A comment or javadoc here explaining why it's deprecated / when it is safe for someone to remove would be helpful in case someone comes across this in the future.